index.html

```diff
@@ -17,148 +17,255 @@
             <a class="logo-container" href="">
                 <img src="images/image_geometry_1logo.png" alt="logo">
             </a>
-
             <nav>
+                <a href="#Hello">Hello</a>
+                <a href="#projects">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
+
                 <a class="a-logo" href=""><i class="fas fa-bars"></i></a>
             </nav>
         </header>
         <section class="headline">
             <div class="name">
                 <h1>Gabriel <br> Nwachukwu</h1>
             </div>
+            <div class="geometry_desktop"></div>
```

Comment:
- [ ] [OPTIONAL] You can diminish the `margin-top` for the button to improve the UX for your website.

![image](https://user-images.githubusercontent.com/60164239/131208118-8c36c761-cfc0-4c39-b3db-f91ad59a6db4.png)